tests/cmake/find_pkg_config/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.0)
+project(find_pkg_config)
+
+message(STATUS "PKG_CONFIG_LIBDIR: $ENV{PKG_CONFIG_LIBDIR}")
+
+find_package(PkgConfig REQUIRED)
+
+pkg_check_modules(EGL REQUIRED egl)
```

Comment:
Maybe add a comment here: `Emscripten ships .pc files for egl/glesv2/sdl.  Check that they can all be located correctly`

Reply:
Added a message explaining what the script is trying to do. I reworded it a bit to not contain the list of packages, so that adding a new package won't require modifying the message on top of everything else. I guess we could just go with a full-pledged list and then iterate, create some nice message and so on, but I feel like that'd be an overkill for a test and I'd rather have it dumb simple.